guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -206,7 +206,7 @@ Sets the declared block or container element as a child of another element in th
 ### remove {#fedg_layout_xml-instruc_ex_rmv}
 
 `<remove>` is used only to remove the static resources linked in a page `<head>` section.
-For removing blocks or containers, use the `<remove>` attribute for [`<referenceBlock>` and `<referenceContainer>`](#fedg_layout_xml-instruc_ex_ref).
+For removing blocks or containers, use the `remove` attribute for [`<referenceBlock>` and `<referenceContainer>`](#fedg_layout_xml-instruc_ex_ref).
```

Comment:
@hgajjar  The incorrect syntax for `<remove>` is also in the v2.1 file,  (guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md).  Can you add the correction to that file as well?